src/beast/beast/module/core/native/linux_Files.cpp

```diff
@@ -145,9 +144,9 @@ File File::getSpecialLocation (const SpecialLocationType type)
 class DirectoryIterator::NativeIterator::Pimpl
 {
 public:
-    Pimpl (const File& directory, const String& wildCard_)
+    Pimpl (const File& directory, const String& _wildCard)
```

Comment:
@scottschurr the one in develop is a little bit behind, I'm using this:
https://github.com/vinniefalco/rippled/blob/style/doc/CodingStyle.md